test/command.bats

```diff
@@ -49,3 +49,28 @@ load helpers
 	[ "$status" -ne 0 ]
 	[[ "$output" =~ "value out of range" ]]
 }
+
+@test "invalid log max" {
+	run ${CRIO_BINARY} --log-size-max foo
```

Comment:
You still have to provide the conmon path to make the test succeed:
```suggestion
	run ${CRIO_BINARY} --conmon "$TESTDIR/conmon" --log-size-max foo
```

Reply:
thanks, done